### PR TITLE
Changes to debugger addon

### PR DESCRIPTION
**Pull request by apple502j**
Read commit messages for details.

image
- [ ] after merging make sure to copy-paste the userscript

### CODE REVIEW

File: addons/debugger/userscript.js [hunk 1]
```diff
@@ -97,6 +108,11 @@ export default async function ({ addon, global, console, msg }) {
     myFlash.block = block;
 
     function _flash() {
+      if (!myFlash.block.svgPath_) {
+        myFlash.timerID = count = 0;
+        flashOn = true;
+        return;
+      }
```
Comment (TheColaber) [inline addons/debugger/userscript.js:115]: This code is taken from devtools. is it worth bringing this change there?

Comment (apple502j) [reply addons/debugger/userscript.js:115]: Gonna check that, bc that means devtools can crash...

Comment (apple502j) [reply addons/debugger/userscript.js:115]: Cannot repro crash, so not gonna fix in this PR.

File: addons-l10n/en/debugger.json [hunk 2]
```diff
@@ -5,5 +5,12 @@
   "debugger/clear": "Clear",
   "debugger/close": "Close",
   "debugger/enter-format": "Enter export format:",
-  "debugger/unpause": "Unpause"
+  "debugger/unpause": "Unpause",
+  "debugger/deleted-sprite": "(deleted sprite)",
+  "debugger/deleted-sprite-desc": "The sprite is deleted, so you cannot find the source.",
```
Comment (WorldLanguages) [inline addons-l10n/en/debugger.json:10]: Unnecessary

File: addons-l10n/en/debugger.json [hunk 3]
```diff
@@ -5,5 +5,12 @@
...
+  "debugger/clone-of": "Clone of {spriteName}",
+  "debugger/clone-desc": "This message is logged by a clone.",
```
Comment (WorldLanguages) [inline addons-l10n/en/debugger.json:12]: Unnecessary

**Comment by WorldLanguages** [comment]
@mxmou Commit above ^ (CSS `!important`) because dark-www made the card title color gray.

**Comment by mxmou** [comment]
> @mxmou Commit above ^ (CSS `!important`) because dark-www made the card title color gray.

`!important` works, but wouldn't setting the color on the card itself instead of the heading work as well?

**Comment by WorldLanguages** [comment]
Yes. Apple decided to go this way :P But sure that's also valid

**Pull request merged**